### PR TITLE
A new rule for a manual check

### PR DESCRIPTION
**Pull request by HelenBurge**
This is basic and a starting point for the first manual test rule.

Please review and see if examples need updating. It tests WCAG 1.1.1 and if an image has a meaningful alternative text.

Closes issue(s):

Need for Call for Review:
This will require a 2 weeks Call for Review 

---

### **When creating PR:**

- [ ] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ ] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [ ] Add yourself (and co-authors) as "Assignees" for PR.
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/image-meaningful-alternative-text.txt [hunk 1]
```diff
@@ -0,0 +1,248 @@
+---
+id:
+name: Image has meaningful alternative text
+rule_type: atomic
+description: |
+  This rule checks that each image with alternative text has a description that is
```
Comment (Jym77) [inline _rules/image-meaningful-alternative-text.txt:6]: Missing end of the sentence.

File: _rules/image-meaningful-alternative-text.txt [hunk 2]
```diff
@@ -0,0 +1,248 @@
+---
+id:
+name: Image has meaningful alternative text
+rule_type: atomic
+description: |
+  This rule checks that each image with alternative text has a description that is 
+accessibility_requirements:
```
Comment (Jym77) [inline _rules/image-meaningful-alternative-text.txt:7]:
```suggestion
```

File: _rules/image-meaningful-alternative-text.txt [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    -  Helen Burge
+  previous_authors:
+    -
```
Comment (Jym77) [inline _rules/image-meaningful-alternative-text.txt:32]:
```suggestion
```

**Review by Jym77 — COMMENTED** [review]
I'm not really sure how the rule differs from [Image accessible name is descriptive](https://www.w3.org/WAI/standards-guidelines/act/rules/qt1vmo/proposed/) 🤔 
Notably in what it tries to achieve. What are the cases that this new rule catch and the old one didn't?

**Comment by HelenBurge** [comment]
> I'm not really sure how the rule differs from [Image accessible name is descriptive](https://www.w3.org/WAI/standards-guidelines/act/rules/qt1vmo/proposed/) 🤔 Notably in what it tries to achieve. What are the cases that this new rule catch and the old one didn't?

Yep - I missed that one so will close this PR and start again :)

**Pull request closed without merging**